assets/css/print.css

```diff
@@ -0,0 +1,46 @@
+body,
```

Comment:
Okay, checked with the other Jekyll folks over here and they said to try adding the frontmatter section (with no frontmatter) before this line and see if that helps Jekyll figure out it should generate the CSS file. So it should look like:
```
---
---
body,
```
Does that look right @shawnbot?